backend/app/api/v1/file.go

```diff
@@ -786,7 +786,7 @@ func (b *BaseApi) Keys(c *gin.Context) {
 // @Tags File
 // @Summary Read file by Line
 // @Param request body request.FileReadByLineReq true "request"
-// @Success 200
+// @Success 200 {object} response.FileLineContent
 // @Security ApiKeyAuth
 // @Security Timestamp
 // @Router /files/read [post]
```

Comment:
1. The first line of the `ListFiles` function should be changed to use a struct literal instead of casting.
```go
files, err := fileService.GetFiles(req)
```

2.`helper.ErrorWithDetail(c, constant.CodeErrInternalServer, constant.ErrTypeInternalServer, err)` can be simplified to `helper.FailWithDetail(c, "internal server error", err)`
```go

err = fileService.GetFileList(req)
if err != nil {
        helper.FailWithDetail(c, "failed to get file list", err)
        return
}

helper.SuccessWithData(c, fileList)

```

3. The change from using `constant.ErrTypeInternalServer` to just stating `"internal server error"` is more explicit and cleaner.

4. In the `Keys` function, if there are multiple keys that need to be returned, it might be better to have those values stored in an array or map before passing them to `SuccessWithData`.

In summary, these changes streamline some boilerplate code such as logging errors or formatting responses and make them easier to read and maintain.